classes/migration/upgrade/v3_5_0/InstallEmailTemplates.php

```diff
@@ -28,11 +28,23 @@ protected function getEmailTemplateKeys(): array
     {
         return [
             'CHANGE_EMAIL',
+            'SUBMISSION_SAVED_FOR_LATER',
+            'SUBMISSION_NEEDS_EDITOR',
+            'REVIEW_COMPLETE',
+            'REVIEW_EDIT',
         ];
     }
 
     public function up(): void
     {
+        // remove the carried-over notifications template rows
+        DB::table('email_templates_default_data')
+            ->where('email_key', 'NOTIFICATION')
+            ->delete();
+        DB::table('email_templates')
+            ->where('email_key', 'NOTIFICATION')
+            ->delete();
+
```

Comment:
@Hafsa-Naeem I was looking to forward the changes of https://github.com/pkp/pkp-lib/issues/11603 to main, I noticed that these changes were not forward ported to main yet as https://github.com/pkp/pkp-lib/blob/main/classes/migration/upgrade/v3_5_0/InstallEmailTemplates.php . Wondering any reason this has been waiting for to be merged ? But the OJS PR is merged .